dev/service-registry/client-core/src/main/scala/com/lightbend/lagom/devmode/internal/registry/package.scala

```diff
@@ -2,7 +2,7 @@
  * Copyright (C) Lightbend Inc. <https://www.lightbend.com>
  */
 
-package com.lightbend.lagom.internal
+package com.lightbend.lagom.devmode.internal.registry
```

Comment:
```suggestion
package com.lightbend.lagom.devmode.internal
```
It's [package object](https://docs.scala-lang.org/tour/package-objects.html) 😉

Reply:
nice :)